ludwig/schema/features/binary_feature.py

```diff
@@ -67,15 +66,24 @@ class GBMBinaryInputFeatureConfig(BinaryInputFeatureConfig):
     )
 
 
+@DeveloperAPI
+@gbm_defaults_config_registry.register(BINARY)
+@ludwig_dataclass
+class GBMBinaryDefaultsConfig(BinaryInputFeatureConfigMixin):
```

Comment:
Should be `BinaryInputFeatureConfig`, not the mixin. Otherwise you're missing `type`.